client/my-sites/posts/post-controls.jsx

```diff
@@ -45,7 +46,7 @@ const getAvailableControls = props => {
 	// Main Controls (not behind ... more link)
 	if ( userCan( 'edit_post', post ) ) {
 		controls.main.push( {
-			className: 'edit',
+			className: 'edit' + ( current === 'edit' ? ' is-active' : '' ),
```

Comment:
Yes, right, it isn't a hard rule though.
I'm not sure if using`classnames` could bring better implementation in terms of simplicity, efficiency, etc.

```es6
  className: 'edit' + ( current === 'edit' ? ' is-active' : '' ),
```

```
  className={ className( 'edit', { 'is-active': current === 'edit' } )
```

Honestly, I prefer the first block that the second one. Change isn't a big deal either.